file_paths.md

```diff
@@ -56,8 +56,8 @@ The files annotated by an '*' indicate that they are created by package installa
         mco@ *                            -> /opt/puppetlabs/puppet/bin/mco
         puppet@ *                         -> /opt/puppetlabs/puppet/bin/puppet
 
-    /opt/puppetlabs/facter
-        facts.d                           # external facts directory (not pluginsync'ed)
+    /opt/puppetlabs/facter *
+        facts.d *                         # external facts directory (not pluginsync'ed)
```

Comment:
Yes, we already create this on Windows at `C:\ProgramData\PuppetLabs\facter\facts.d`

https://github.com/puppetlabs/puppet-specifications/blame/master/file_paths.md#L152-L153